code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling.dm

```diff
@@ -10,20 +10,22 @@
 	desc = "It never stays still for long."
 	icon_state = "spiderling"
 	icon_dead = "spiderling_dead"
+	density = FALSE
 	faction = list(FACTION_SPIDER)
 	speed = 1
 	move_resist = INFINITY // YOU CAN'T HANDLE ME LET ME BE FREE LET ME BE FREE LET ME BE FREE
 	speak_emote = list("hisses")
 	initial_language_holder = /datum/language_holder/spider
 	basic_mob_flags = FLAMMABLE_MOB | DEL_ON_DEATH
 	mob_size = MOB_SIZE_TINY
+	var/menu_description = "Normal spiderling."
```

Comment:
This should have a documenting comment.
However, as I advised you before, I won't merge this while the menu displays spiderlings. The egg selection menu should display the grown spider people are choosing, not the spiderling.

Reply:
I personally disagree I think it's better to show people what they will actually become other than what they will become so players understand they will be spawning as spiderlings the menu description gives the abilities and how the spider looks is purely cosmetic 

It's an egg it produces spiderlings not fully grown spiders

Reply:
Yes players should see what they are actually becoming: the adult spider. This is useful information, the spiderling is not useful information.
Unless you can convince another maintainer otherwise I will DNM this until you fix this.